app/assets/stylesheets/ama_layout/layout_components/option-toggle.scss

```diff
@@ -0,0 +1,52 @@
+.option-toggle{
+  &__item{
+    @include clearfix;
+    margin: 1px 0;
+    display: block;
+    height: inherit;
+    line-height: 1;
+    background-color: $white;
+    color: $cerulean;
+    position: relative;
+
+    &:nth-child(even){
+      background-color: $ash;
+    }
+
+    &.visible{
+      background-color: $brand-blue-light;
+      color: $white;
+
+      .option-toggle__item-icon{
+        background-color: $brand-blue-light;
+        color: $white;
+      }
+    }
+  }
+
+  &__item-label{
+    display: block;
```

Comment:
Shouldn't need display: block, or float since the positioning is set to absolute through the vertical-center include. Also shouldnt need the line-height to center text since vertical align is doing that too. Will most likely need to change the left margin once these are removed.